components/applications-service/pkg/storage/postgres/supervisors.go

```diff
@@ -0,0 +1,13 @@
+package postgres
+
+const (
+	selectSupervisorsTotalCount = `
+SELECT count(*)
+  FROM supervisor;
+`
+)
+
+func (db *Postgres) GetSupervisorsCount() (int32, error) {
+	count, err := db.DbMap.SelectInt(selectSupervisorsTotalCount)
```

Comment:
These queries are very small, I would recommend only creating constants when the query needs to be written down with format so that it is easier to understand it.

```suggestion
	count, err := db.DbMap.SelectInt("SELECT count(*) FROM supervisor")
```

Same for the other two above. (`service` and `service_group`)